xcode/Subconscious/Shared/Components/FirstRun/FirstRunProfileView.swift

```diff
@@ -13,18 +13,34 @@ struct FirstRunProfileView: View {
     @Environment(\.colorScheme) var colorScheme
     
     var body: some View {
+        let did = Did(app.state.sphereIdentity ?? "") ?? Config.default.subconsciousGeistDid
         NavigationStack {
-            VStack {
+            VStack(spacing: AppTheme.padding) {
                 Spacer()
+                StackedGlowingImage(
+                    width: 180,
+                    height: 180
+                ) {
+                    GenerativeProfilePic(
+                        did: did,
+                        size: 180
+                    )
+                }
+                
+                Text("This is your sphere. It stores your notes.")
+                    .foregroundColor(.secondary)
+                Spacer()
+                Text("What should we name it?")
+                        .foregroundColor(.secondary)
                 VStack(alignment: .leading, spacing: AppTheme.unit4) {
                     ValidatedTextField(
-                        placeholder: "nickname",
+                        placeholder: "my-nickname",
```

Comment:
I personally prefer "nickname". `my-nickname` seems too abstract to me, and might encourage people to write `my-subconscious` or something like that.